procedures/meeting-observers.md

```diff
@@ -30,6 +30,12 @@ Intentional disruption of the meeting will result in an observer being kicked fr
 
 It is expected that, by default, those joining as meeting observers will follow the above guidelines. However, there may be times where the council will invite guests to participate beyond the role and guidelines of meeting observers. This may happen ahead of the meeting time or, at the discretion of the meeting participants, during the meeting itself. Under these circumstances, adherence to meeting observer guidelines is not expected.
 
+### Discretion
+
+Observers may share what they observe in the meeting.
```

Comment:
Do we want to state that a council member may explicitly state a topic is sensitive and observers are then not allowed to share what is discussed? It seems dubious to put the decision power of whether a topic is sensitive or not fully in the hands of the observer.

Reply:
I have mixed feelings here. I left "must use discretion" pretty undefined, but I would think honoring requests like "hey, we're not quite ready to share this widely, could you please keep it quiet for now" would fall under that. In practice, the observers we've had have been well-known to most if not all the council, and we have a certain amount of trust built up with them. I'd rather rely on that trust that adding more policy language.

For things that really need to be kept to council-ears-only, the council shouldn't be talking about them in front of observers. I see the part about discretion as asking observers for some grace if the conversation accidentally drifts into something that should have been in a private session. But perhaps a useful side effect of a policy like this is it'd help us do better about keeping private things to closed sessions.